src/cmd/create.go

```diff
@@ -219,6 +221,15 @@ func createContainer(container, image, release, authFile string, showCommandToEn
 
 	pulled, err := pullImage(image, release, authFile)
 	if err != nil {
+		if errors.Is(err, errorNotTerminal) {
+			var builder strings.Builder
+			fmt.Fprintf(&builder, "User interaction required to pull a new image. Stopping now.\n")
+			fmt.Fprintf(&builder, "Run '%s --help' for usage.", executableBase)
```

Comment:
Nitpick: the error could hint at the potential solution.

For example, if we re-use the text from the image download prompt, then:
```
Error: image required to create toolbox container.
The '--assumeyes' option is needed.
Run 'toolbox --help' for usage.
```

This would fit with our pattern for mentioning the immediate cause of the error, followed by the potential solution or the cause of the error.